kubernetes-deployment/bootstrap.py

```diff
@@ -513,6 +513,24 @@ def main():
         logger.info("Clean up job finished")
         return
 
+    if args.action == 'etcdfix':
+
+        logger.info("Begin to fix broken etcd server.")
+
+        node_list_config = load_yaml_file(args.file)
+
+        logger.debug("FIX ETCD on {0}".format(str(node_list_config)))
+
+        if len(node_list_config['machinelist']) != 1:
+
+            logger.error("etcdfix can't fix more than one machine everytime. Please fix them one by one!")
```

Comment:
>etcdfix can't fix more than one machine everytime. Please fix them one by one! [](start = 26, length = 78)

just to double check: if 2 of 3 etcd nodes are down, the script can fix them one by one, right?